lib/src/local_working_copy.rs

```diff
@@ -106,7 +106,7 @@ use crate::working_copy::WorkingCopyStateError;
 
 #[cfg(unix)]
 type FileExecutableFlag = bool;
-#[cfg(windows)]
+#[cfg(any(windows, target_os = "wasi"))]
```

Comment:
I can comment this in the repo if wanted, but wasi doesn't have a concept of file permission flags right now https://github.com/WebAssembly/wasi-filesystem/blob/main/wit/types.wit

Reply:
Yes, please do leave a comment. Is there a ticket or some kind of ideas for this? It would be enough to just leave a comment to a relevant ticket i.e. `// wasi symlink support: https://github.com/webassembly/wasi-filesystem/issue/...`

Reply:
I haven't found any workstreams trying to add executable flag support for wasi.

I would assume executable files would be added in a separate interface. I'm pretty sure wasi doesn't have the ability to execute another process right now in the current spec, marking a file as executable wouldn't be meaningful.

I'll add a comment as well for this line, if I do run into an issue link I'll append that.